src/content/transformers/candidateContentPromptsTransformer.ts

```diff
@@ -0,0 +1,26 @@
+import { Logger } from '@nestjs/common';
+import {
+  Transformer,
+  AIContentTemplateRaw,
+  CandidateContentPrompts
+} from '../content.types';
+import { camelCase } from 'lodash';
+
+const logger = new Logger('CandidateContentPromptsTransformer');
+
+export const candidateContentPromptsTransformer: Transformer<
+  AIContentTemplateRaw,
+  CandidateContentPrompts
+> = (templates: AIContentTemplateRaw[]): CandidateContentPrompts[] => {
+  const result = templates.reduce<CandidateContentPrompts>((acc, template) => {
+    if (template.data.name && template.data.content) {
+      const name = camelCase(template.data.name);
```

Comment:
https://www.npmjs.com/package/just-camel-case

Reply:
There's also no need to create and store this in another variable if all you're doing is using it in one place on the next line.

Just do.. 

`acc[camelCase(template.data.name)]`